README.md

```diff
@@ -24,12 +24,34 @@ ChatGPT uses your private conversations history to train its models. [OpenAI dis
 
 # Table of Contents
 
+- [Configuration](#configuration)
 - [Deploy the app on Render](#deploy-the-app-on-render)
 - [Deploy the app on Fly.io](#deploy-the-app-on-fly)
 - [Contribute as a developer](#contribute-as-a-developer)
 - [Understanding the Docker configuration](#understanding-the-docker-configuration)
 - [Changelog](#changelog)
 
+# Configuration
+
+Following features can be configured with environment variables:
+
+- `REGISTRATON_FEATURE` - Set to `false` to disable registration.
+  Default is `true`.
+- `AUTHENTICATION_APPROACH` - Set to `google` to use Google OAuth for authentication, `http_header` to use HTTP header authentication, or `password` to use password authentication.
+  Default is `password`.
```

Comment:
It’d be nice to be able to support multiple: google & password, for example. Would it ever make sense to support all three or if you are using HTTP_HEADER is that necessarily the only way to auth?

Reply:
Password and Google could co-exist. HTTP Auth would not make sense to co-exist with anything else

In many cases, one would like to disable password auth when one has Google Auth enabled

Reply:
I will change it, as described above

Reply:
I updated the README and refactored the feature flags